crates/iota/src/name_commands.rs

```diff
@@ -968,7 +968,6 @@ impl std::fmt::Display for NameCommandResult {
                         nft.id().to_string(),
```

Comment:
Bit further up the "image URL" header should also be removed

Reply:
https://github.com/iotaledger/iota/pull/6688/commits/01a3d002d81ecdabc646d77621928a8fdd96a927